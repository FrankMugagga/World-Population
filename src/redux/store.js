import { configureStore } from '@reduxjs/toolkit';
import countriesReducer from '../features/countries/countriesSlice';

const store = configureStore({
  reducer: {
    countries: countriesReducer,

  },

});

export default store;
