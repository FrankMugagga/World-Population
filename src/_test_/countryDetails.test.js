import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import '@testing-library/jest-dom';
import CountryDetails from '../features/countries/countryDetails/DetailsCountry';

describe('Testing the country details', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <CountryDetails />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('That jest works', () => {
    expect(true).toBe(true);
  });
});
