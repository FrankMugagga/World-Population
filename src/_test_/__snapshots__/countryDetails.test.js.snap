// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the country details matches snapshot 1`] = `
<div
  className="countDetails"
/>
`;
